app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/CollectionServiceStarter.java

```diff
@@ -22,6 +22,16 @@
 
     private final static String TAG = CollectionServiceStarter.class.getSimpleName();
 
+
+    public static boolean isWifiandBTWixel(Context context) {
+        SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(context);
+        String collection_method = prefs.getString("dex_collection_method", "WifiBlueToothWixel");
+        if(collection_method.compareTo("WifiBlueToothWixel") == 0) {
```

Comment:
please use: prefs.getString("dex_collection_method", "BluetoothWixel"));
The default should be the same across all project.